packages/dso-toolkit/src/components/progress-bar/progress-bar.mixins.scss

```diff
@@ -23,5 +18,10 @@
       height: 100%;
       width: 0%;
     }
+
+    + span {
```

Comment:
Dit is eigenlijk de enige change die te rechtvaardigen is: De andere changes gaan allemaal over nodes in Shadow DOM, daar was het ID toch al uniek

Reply:
Daar ben ik het wel mee eens, maar willen we niet gewoon helemaal uitsluiten dat er op id's wordt gestijlt? Daar ben ik eigenlijk wel van uit gegaan